README.md

```diff
@@ -72,13 +73,18 @@ This project is built with React
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO
+LIVE DEMO
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://React-Capstone-Project-oin4.onrender.com)
+- [Live Demo Link](https://reactcapstone-project.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- [Loom Link](https://www.loom.com/share/cbf59121ab274218ba4935f3d2f8833f)
```

Comment:
-You have done a great job 💪 presenting your project. However,🙆‍♂️ you are still missing the highlighting of a piece of code you are interested in. Instead of only talking about the piece of code you like, it's good 👍to show the actual code from your text editor or GitHub. This is an important part of the video presentation and will demonstrate your efforts in the project.